index.html

```diff
@@ -171,7 +171,10 @@
 function matrixToString(m)
 {
     var r = m.elements;
-    var s = '[ '+toReal(r[0])+', '+toReal(r[4])+', '+toReal(r[8])+';\n'+toReal(r[1])+', '+toReal(r[5])+', '+toReal(r[9])+';\n'+toReal(r[2])+', '+toReal(r[6])+', '+toReal(r[10])+' ]';
+    var s = 
+         '[ ' +pmOneToStr(r[0],7)+', '+pmOneToStr(r[4],7)+', '+pmOneToStr(r[8],7)+';\n'
```

Comment:
Since you only use this function with argument "7", you can drop the argument and make 7 default.

Reply:
I changed name to 'toFixedWidth'